src/StorageSync/StorageSync/StorageSyncService/SetStorageSyncServiceCommand.cs

```diff
@@ -123,8 +123,8 @@ public class SetStorageSyncServiceCommand : StorageSyncClientCmdletBase
             Mandatory = false,
             HelpMessage = HelpMessages.StorageSyncServiceIdentityTypeParameter)]
         [ValidateSet(StorageSyncServiceIdentityType.systemAssigned,
-            StorageSyncServiceIdentityType.userAssigned,
-            StorageSyncServiceIdentityType.systemAssignedUserAssigned,
+            //StorageSyncServiceIdentityType.userAssigned,
```

Comment:
this is breaking change removing values from validate set

Reply:
We never supported this value